lib/analytics/page.rb

```diff
@@ -0,0 +1,72 @@
+module Analytics
+  class Page
+    def self.increment(page_id, new_member:)
+      new(page_id).increment_actions(new_member: new_member)
+    end
+
+    def initialize(page_id)
+      @page_id = page_id
+    end
+
+    def total_actions(new_members: false)
```

Comment:
why don't we have one method called `total_actions` and one called `total_new_members`? I understand making it a parameter for code reuse in the private functions, but I think it makes the API less clear.